BusRoutes/View/RouteDetails.xaml

```diff
@@ -0,0 +1,34 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<ContentPage xmlns="http://xamarin.com/schemas/2014/forms" xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml" x:Class="BusRoutes.View.RouteDetails" Title="{Binding Title}">
+	<ContentPage.Content>
+	<StackLayout>			
+			<ActivityIndicator IsRunning="{Binding IsLoading}" IsVisible="{Binding IsLoading}" />							
+			<Button Text="Bus Stop" FontSize="20" Clicked="BusStopButton_OnClick" x:Name="BusStopButton" />
+			<ListView ItemsSource="{Binding Streets}" x:Name="StreetListView" ItemSelected="StreetListView_OnItemSelected" IsVisible="false">
+				<ListView.ItemTemplate>
+					<DataTemplate>
+						<ViewCell>
+							<StackLayout>								
+								<Label Text="{Binding Name}" />
+							</StackLayout>
+						</ViewCell>
+					</DataTemplate>
+				</ListView.ItemTemplate>
+			</ListView>
+			<Button Text="Timetable" FontSize="20" Clicked="TimetableButton_OnClick" x:Name="TimetableButton" />
+			<ListView ItemsSource="{Binding DepartureSummaries}" ItemSelected="TimetableListView_OnItemSelected" RowHeight="100" x:Name="TimetableListView" IsVisible="false" >
```

Comment:
Avoid long lines in xaml, try to put line breaks for each property, this helps both development and code review